cookbook/sortAbsoluteInstants.mjs

```diff
@@ -0,0 +1,28 @@
+/**
+ * getSortedInstants will sort an array of strings (each of which is parseable as a Temporal.Absolute and may or may not include an IANA time zone name)
+ * by the corresponding absolute time (e.g., for presenting global log events sequentially).
+ *
+ *
+ * @param {string[]} parseableAbsoluteStrings This is a Date instance
+ * @param {boolean} reverse accending or descending order
+ * @returns {Temporal.Absolute[]} Temporal.Absolute instance
+ */
+function getSortedInstants(parseableAbsoluteStrings, reverse = false) {
+    const absoluteTimes = parseableAbsoluteStrings.map(v => {
+        return Temporal.Absolute.from(v);
+    });
+
+    const sortedAbsoluteTimes = absoluteTimes.sort((a, b) => {
+        return a.compare(b);
```

Comment:
Same thing as in the other pull request, this would need to be `absoluteTimes.sort(Temporal.Absolute.compare)` to fix the "compare is not a function" problem